src/main/scala/com/microsoft/hyperspace/actions/RefreshAction.scala

```diff
@@ -17,61 +17,15 @@
 package com.microsoft.hyperspace.actions
 
 import org.apache.spark.sql.SparkSession
-import org.apache.spark.sql.types.{DataType, StructType}
 
-import com.microsoft.hyperspace.HyperspaceException
-import com.microsoft.hyperspace.actions.Constants.States.{ACTIVE, REFRESHING}
 import com.microsoft.hyperspace.index._
 import com.microsoft.hyperspace.telemetry.{AppInfo, HyperspaceEvent, RefreshActionEvent}
 
-// TODO: This class depends directly on LogEntry. This should be updated such that
-//   it works with IndexLogEntry only. (for example, this class can take in
-//   derivedDataset specific logic for refreshing).
 class RefreshAction(
```

Comment:
Looks like you removed the existing comment on this class. Can you add the new class documentation?

Reply:
Thanks! I see you updated the class doc. I'll add my comment there.

Reply:
TODO is moved to the base class. I added class doc.